doc/build/changelog/unreleased_20/10904.rst

```diff
@@ -0,0 +1,7 @@
+.. change::
+    :tags: usecase, postgresql
+    :tickets: 10904
+
+    Support the ``USING <method>`` option for PostgreSQL ``CREATE TABLE`` to
+    specify the access method to use to store the contents for the new table.
+    Pull request courtesy Edgar Ramírez-Mondragón.
```

Comment:
**Michael Bayer** (zzzeek) wrote:

this could use a ..seealso: that links to the doc

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5114

Reply:
Thanks for the review.

* I've added docs for the new option, added a `versionadded` note, and reordered the options to be alphabetical: feaa7b7382a73fca247182693eae09b6afed6104
* I've added a `..seealso` directive to the changelog note: 85f232a303a5543725dac42206cb2395fc34109e

I didn't find how to handle the backport PR in the dev docs. Should it be a cherry-pick that targets the `rel_2_0` branch?

Reply:
I also took the liberty of creating https://github.com/sqlalchemy/sqlalchemy/issues/10909

Reply:
> 
> I didn't find how to handle the backport PR in the dev docs. Should it be a cherry-pick that targets the `rel_2_0` branch?

That can be done by me or mike, no worries

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5114